Remotion/Web/Development.WebTesting/WebTestConfigurationFactory.cs

```diff
@@ -44,6 +47,13 @@ public class WebTestConfigurationFactory
     /// </summary>
     protected const string LatestTestedChromeVersion = "76";
     
+    /// <summary>
+    /// Represents the latest version of Edge verified to be compatible with Selenium WebDriver.
+    /// Higher Edge versions are not guaranteed to work with the currently dependent MSEdgeDriver. 
+    /// In order to achieve a stable testing environment a standalone Edge with a matching version should be used.
+    /// </summary>
+    protected const string LatestTestedEdgeVersion = "77";

```

Comment:
unless there is a reason, I think we should keep the vnumbers for edge and chrome synchronous.